Desktop/python_exercise/recursion_sum.py

```diff
@@ -0,0 +1,10 @@
+numbers = [1, 2, 3, 4, 5]
+
+def sum_recursion(nums, index):
+    if index == 0:
+        return nums[index]
+    else:
+        return nums[index] + sum_recursion(nums, index - 1)
+length =  len(numbers) - 1
+total_sum = sum_recursion(numbers,length)
```

Comment:
Let's change this logic.
Only pass the numbers in the function and it should sum all the numbers of list. Remove length argument from function.